bcda/auth/models.go

```diff
@@ -63,3 +68,56 @@ func GetACOByClientID(clientID string) (models.ACO, error) {
 	}
 	return aco, err
 }
+
+/*
+ GenerateSystemKeyPair creates a keypair for a system, based on the provided client ID.
+ The public key is saved to the database and the private key is returned.
+*/
+func GenerateSystemKeyPair(clientID string) (string, error) {
```

Comment:
To use the key, we seem to be having to convert the `string` to a `[]byte`.  Would that be a better return type?

Reply:
I was thinking about that. The CLI command function could convert the `[]byte` to `string` to print it out. 

Before I make the change, any preferences about the return type, @DeirdreHolub or @alexhova ?

Reply:
no preference here!